index.js

```diff
@@ -0,0 +1,4 @@
+const title = document.getElementById("title")
+title.addEventListener("click", function () {
+    alert("You clicked the title!")
```

Comment:
We should make the text of the alert more engaging.  Maybe add more exclamation points!!!